### DISCUSSION TITLE
Do you plan to make a separate details page for Trace?

### DISCUSSION THREAD

**Discussion by alanlvle** [apache/skywalking#8200] [Q&A] [answered]
for example:
get   http://***/trace/201836be9f754f4f80cdc7df037a9008.94.16381536953150001

result:
show trace id details

**Comment by hanahmily** [comment, marked as answer]
we leverage the graphql based [query protocol](https://github.com/apache/skywalking-query-protocol), don't have any plan to switch to a restful style.

**Comment by alanlvle** [comment]
I may not express it clearly. What I mean is to access the traceId details page through this url, not the API interface to get data.
Now the trace page has been accessed using graphql.
example:
![image](https://user-images.githubusercontent.com/25681693/143974605-9ecdfb10-7571-4162-8be6-a888eab3a24c.png)
we can use vue method this.$route.params get traceId realization this detail page.
this detail page Sharing is very convenient。pinpoint zipkin all have this page.

**Reply by wu-sheng** [reply]
No, we don't have that. APIs are what we exposing only.

**Reply by wu-sheng** [reply]
If you want to share the whole system page, this works, https://demo.skywalking.apache.org/trace?traceid=4df350f5-a1a0-48c0-8fd6-aa4d3f499d04.

**Reply by alanlvle** [reply]
understand，I think the data is enough，Do we plan to implement this page on the existing basis?

**Reply by wu-sheng** [reply]
In SkyWalking, we don't have plan to do anything. All contributors are treated equally, we don't change codes according to requests, we just review and decide whether accept the proposed changed from global developers. 
So, if you want to ask for some changes, bug fixes, enhancements, the only option is you or someone else are willing to contribute this to the upstream community.

**Reply by alanlvle** [reply]
comprehend，I try to do it.thx!
